interpreter/src/main/java/org/qbicc/interpreter/impl/Frame.java

```diff
@@ -1006,7 +1006,7 @@ public Object visit(VmThreadImpl thread, IsGe node) {
         } else if (isFloat64(inputType)) {
             return Boolean.valueOf(unboxDouble(left) >= unboxDouble(right));
         } else if (isInt64(inputType)) {
-            return Boolean.valueOf(unboxLong(left) >= unboxLong(right));
+            return Boolean.valueOf(isSigned(inputType) ? compareLongSigned(left, right, -1) >= 0 : compareLongUnsigned(left, right, -1) >= 0);
```

Comment:
Should we be doing something similar (although much simpler since they can't be hiding pointers) for the rest of the Integer types in the next stanza of the if-then-else?
```
isSigned(inputType) ? Integer.compare(left, right) ... : Integer.compareUnsigned(left, right) ...
```
Or is there an invariant that the types must be signed because of the way the graph is constructed?  Or something else...

Reply:
Yes I think so. I'll do a separate PR for it.